lib/ja_serializer/builder/pagination_links.ex

```diff
@@ -0,0 +1,94 @@
+defmodule JaSerializer.Builder.PaginationLinks do
+  import JaSerializer.Formatter.Utils, only: [format_key: 1]
+
+  @page_number_origin Application.get_env(:ja_serializer, :page_number_origin, 1)
```

Comment:
I'll update the README. If this value changes a recompile of ja_serializer will have to be forced